### PR TITLE
Fix goerli based banner show

### PR DESCRIPTION
**Pull request by sepehr2github**
## Description

- Fix goerli based banner show

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: src/containers/navbar/deprecationBanner.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import {GOERLI_BASED_NETWORKS} from 'utils/constants';
 import {useTranslation} from 'react-i18next';
 
 const DeprecationBanner: React.FC = () => {
-  const [bannerHidden, setBannerHidden] = useState(false);
+  const [bannerHidden, setBannerHidden] = useState(true);
```
Comment (cgero-eth) [inline src/containers/navbar/deprecationBanner.tsx:13]: I suppose the close button should be black. Also clicking on it does not hide the banner
Screenshot 2024-02-15 at 11 38 41

**Review by Barukimang — APPROVED** [review]
Checked this @sepehr2github: banner only appears at testnets, and NOT at mainnets, so all good!

File: src/containers/navbar/deprecationBanner.tsx (hunk 1 above)
Comment (sepehr2github) [reply src/containers/navbar/deprecationBanner.tsx:13]: Fixed!

**Review by cgero-eth — APPROVED** [review]
✅

**Pull request merged**